core/src/main/groovy/org/docToolchain/scripts/asciidoc2confluence.groovy

```diff
@@ -169,7 +169,7 @@ def rewriteMarks (body) {
 // The map is indexed by lower-case title
 def retrieveAllPages = { String spaceKey ->
     // #938-mksiva: added a condition spaceKeyInput is null, if it is null, it means that, space key is different, so re fetch all pages.
-    if (allPages != null && spaceKeyInput == null) {
+    if (allPages != null && spaceKe355yInput == null) {
```

Comment:
i am not sure if this was your intention...

Reply:
Sorry - I must have types in the wrong window.